target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-whw03v2.dts

```diff
@@ -0,0 +1,555 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+    model = "Linksys WHW03 V2 (Velop)";
+    compatible = "linksys,whw03v2", "qcom,ipq4019";
+
+    aliases {
+        led-boot = &led_blue;
+        led-failsafe = &led_red;
+        led-running = &led_green;
+        led-upgrade = &led_red;
+    };
+
+    chosen {
+        bootargs-append = " root=/dev/ubiblock0_0 rootfstype=squashfs ro";
```

Comment:
`rootfstype` and `ro` are probably not needed at all.

Reply:
That's also what I thought, but if I remove them the device does not boot. Sadly, I lost the error message. I'll give it another try.

Reply:
Do you know what is the default bootargs that are passed, it could be that UBIFS for example is passed so it needs to get overriden

Reply:
I went back to extract them:
```
partbootargs=init=/sbin/init rootfstype=ubifs ubi.mtd=rootfs root=ubi0:ubifs rootwait rw
partbootargs2=init=/sbin/init rootfstype=ubifs ubi.mtd=alt_rootfs root=ubi0:ubifs rootwait rw
```

Reply:
Ok, then then overriding them both is required, thanks.
I would just add a comment in the DTS so that nobody tries to "fix" it later

Reply:
done